website/pages/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and Vault's Integrated Storage as storage backends.
+
+Many other options for storage are available with community support - see our [Storage Configuration](/docs/configuration/storage) section for more information.
+
+## Backups & Restores
+
+Due to the highly flexible nature of Vault's potential storage configurations, providing exact guidance on backing up and restoring Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
+
+There's also a big question - what is the error case you're trying to guard against by saving a backup?
+
+### The Big Question - Why Take Backups?
+
+It's important to consider the question of "why take a backup" while developing your ongoing
+backup strategy.
+
+We do not recommend backups as protection against the failure of an individual machine or data
+center. Vault servers can run in clusters. Vault Enterprise supports replicated clusters for
+performance and disaster recovery. With open source features, a Vault cluster can extend across
+multiple availability zones within a region.
+
+As you develop a plan for backing up and restoring Vault, include clustering in your plan to
+address redundancy concerns.
+
+### Vault's Persisted Data
+
+Backups and restores are ideally performed while Vault is offline. If offline backups are not
+feasible, we recommend using a storage backend that supports atomic snapshots (such as
+[Consul](https://www.consul.io/docs/commands/snapshot.html) or [Integrated
+Storage](/docs/commands/operator/raft#snapshot)).
+
+**If your storage backend does not support atomic snapshots, we recommend only taking offline backups.**
+
+We recommend taking backups **before**, but not during, write operations to the `/sys` API
```

Comment:
Do we want to give any explanation for why we recommend this?  e.g. "Even with atomic snapshots, certain Vault operations may consist of multiple writes, and a snapshot taken in the middle of them may result in an inconsistent state and thus a less useful backup."

Reply:
I followed up with Nick and will follow up with Mark, but the short answer is no. Feel free to reach out to me for further discussion.

Reply:
This section may fit better in the "Why take backups" section.

Reply:
I think understanding when to take a backup is more of a "how do I" and less of a "should I" kind of thing - could you elaborate on the connection to the "should I" section?

Reply:
I more read it as you "should" take a backup **before** doing system changing operations. Which i think is more useful than saying don't take a snapshot **during** these operations.